client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+export class CommentDetailAuthor extends Component {
+	static propTypes = {
+		author: PropTypes.object,
+		commentDate: PropTypes.string,
+	};
+
+	state = {
+		isExpanded: false,
+	};
+
+	toggleExpanded = () => {
+		this.setState( { isExpanded: ! this.state.isExpanded } );
+	}
+
+	render() {
+		const {
+			author,
+			commentDate,
+			moment,
+			translate,
+		} = this.props;
+		const { isExpanded } = this.state;
+
+		const classes = classNames( 'comment-detail__author', {
+			'is-expanded': isExpanded,
+		} );
+
+		return (
+			<div className={ classes }>
+				<div className="comment-detail__author-preview">
+					<div className="comment-detail__author-avatar">
+						<img src={ author.avatar_URL } />
```

Comment:
As we chatted about earlier, let's use flat props in our new components.  For example `authorAvatarURL` vs `author.avatar_URL`

This does two things:
1. Guards against future data shape changes
2. Makes the data needs of the component much more explicit